### PR TITLE
Add lightroom IBL, enable ColorGrading by default in gltf_viewer.

### PR DESCRIPTION
**Pull request by prideout**
This IBL was created at Google for SceneViewer, and is also used by `<model-viewer>`.  Before / After below.

double

The non-legacy ACES is now default, it looks much better.  I also tested this IBL with WebGL and Android.

lightroom

### CODE REVIEW

**Comment by romainguy** [comment]
Looks great! Did you try to match the intensity as well?

File: samples/gltf_viewer.cpp [hunk 1]
```diff
@@ -96,9 +96,9 @@ struct App {
     } scene;
 
     struct ColorGradingOptions {
-        bool enabled = false;
+        bool enabled = true;
         int quality = static_cast<int>(ColorGrading::QualityLevel::MEDIUM);
-        int toneMapping = static_cast<int>(ColorGrading::ToneMapping::ACES_LEGACY);
+        int toneMapping = static_cast<int>(ColorGrading::ToneMapping::ACES);
```
Comment (romainguy) [inline samples/gltf_viewer.cpp:101]: I am not sure that's what we want to do here. This tone mapper is darker which will create bigger mismatches with other renderers and may make comparisons more difficult.

Comment (prideout) [reply samples/gltf_viewer.cpp]: Non-legacy looks better but yeah good point -- I changed it back.

Comment (romainguy) [reply samples/gltf_viewer.cpp]: It shouldn't: the only difference between legacy and non-legacy is exposure. Legacy is a slightly brighter version of non-legacy. You can get the same result by just changing your camera exposure and/or your light intensities.

Comment (romainguy) [reply samples/gltf_viewer.cpp]: The exposure difference is 1/0.6.

**Comment by prideout** [comment]
> Looks great! Did you try to match the intensity as well?

Just now made web match better with desktop, but I didn't try to match everything with model-viewer, they do stuff like setExposureDirect(1.0) which I'm not sure is best practice.

File: samples/gltf_viewer.cpp (hunk 1 above)
Comment (prideout) [reply samples/gltf_viewer.cpp]: Ah interesting -- understood!  I think it looks pretty good right now actually, I think I'll go ahead and merge this.

**Pull request merged**